nimare/meta/ale.py

```diff
@@ -227,7 +227,7 @@ def _compute_null_analytic(self, ma_maps):
 
             # Reset histogram and set probabilities. Use at() because there can
             # be redundant values in score_idx.
-            ale_hist = np.zeros((inv_step_size,))
+            ale_hist = np.zeros(ale_hist.shape)
```

Comment:
Is this a fix or a refactor? I.e., is `inv_step_size` equal to the length of `ale_hist`?

In any case, it's a clearer version, so I'd like to use it if it's correct. I just want to make sure that the summary is accurate.

Reply:
`inv_step_size` was 10,000 by default whereas ale_hist.shape was 10,0010. This only made a difference when I was using the MKDA kernel with ALE where the `ale_score` could be 10,001, which is is out of range for `inv_step_size`. But this was just so there wouldn't be an error in the code, not necessarily because this is correct, @tyarkoni, is it better to use original size of `ale_hist` or should we use `inv_step_size` instead?

Reply:
If this is a reflection of my recent changes, I don't think I thought very deeply about this, so if they break things, I see no problem in reverting.